base/float.jl

```diff
@@ -88,18 +88,26 @@ exponent_mask(::Type{Float64}) =    0x7ff0_0000_0000_0000
 exponent_one(::Type{Float64}) =     0x3ff0_0000_0000_0000
 exponent_half(::Type{Float64}) =    0x3fe0_0000_0000_0000
 significand_mask(::Type{Float64}) = 0x000f_ffff_ffff_ffff
+mantissa_width(::Type{Float64}) = UInt64(52)
```

Comment:
Note that `mantissa_width` and `exponent_width` are the same as the existing functions `significand_bits` and `exponent_bits`, respectively, but return unsigned values. How annoying would it be to reuse the existing functions and convert to unsigned values as needed?

Reply:
Seems fine I guess. The conversions should be free no?

Reply:
Actually the conversions aren't even necessary since it's all constpropd anyway.